docs/instance.md

```diff
@@ -31,9 +67,55 @@ Then, it creates some folders:
 - `/Documents/Music`, with the id `io.cozy.folders-music`
 - `/Documents/Videos`, with the id `io.cozy.folders-videos`
 
-The ids are forced to known values. So, even if these folders are moved or
-renamed, they can still be found for the permissions.
+**The ids are forced to known values :**  even if these folders are moved or
+renamed, they can still be found for the permissions. 
+
+**The names are localized :** If a locale is provided through the CLI, the folders will be created with 
+names in this locale. Otherwise, theses folders will be created in english and renamed to localized name the first time the locale is set (during onboarding).
+
+Then it creates the basic settings 
+
+- `owner_email` if an email was provided through the CLI
+- `locale` if a locale was provided through the CLI
+- `home_app` URL of the app to use as home, github.com/cozy/cozy-home by default
+- `onboarding_app` URL of the app to use as onboarding, github.com/cozy/cozy-onboarding by default
+- `market_url` URL of the market place to fetch manifests of installable applications.
+
+Settings are created as named id in the `$PREFIX/io.cozy.settings` database.
+During onboarding, the fields will be prefilled with these value if they were provided.
+
+Finally, default applications are installed in the following order : 
+
+- the `home` and `onboarding` applications are installed according to provided URL or cozy defaults. 
+- If the environment is set to `dev`, some devtools are installed
+- If the `apps` CLI param is given, all these apps are installed
+
 
-Finally, the default application for the current environment are installed.
+--------------------------------------
 
-**TODO** explain that the folder names will be localized, and how
+
+Renaming
+--------
+
+An instance is renamed through the command line.
+
+```sh
+$ cozy-stack rename <olddomain> <newdomain>
+```
+
+Renaming an instance only change the HostName in global/instances base.
+
+TODO : should the map Host -> instance be partially cached in cozy-stack RAM ? 
```

Comment:
Globally, we will have cache for the cozy-stack. For single host, the cache will be in the cozy-stack RAM. For multiple servers, the cache will be in redis, so it can be shared between the servers. I don't think we should mention cache in this document.